examples/BlazorServerApp/UserAgentContextProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BlazorServerApp
+{
+    public class UserAgentContextProvider
```

Comment:
```suggestion
    public class UserAgentContext
```

Reply:
I wouldn't consider this a provider. It has no methods. It doesn't do anything. Rather, this is a data model. As such, I suggest the rename.